types/events.go

```diff
@@ -10,22 +10,17 @@ import (
 
 // Reserved event types
 const (
-	EventBond              = "Bond"
```

Comment:
omg lol! ancient events. That said we do want events for valset changes: https://github.com/tendermint/tendermint/issues/1916

Reply:
we'll reintroduce them in #1916 then